templates/demo-store/app/lib/session.server.ts

```diff
@@ -34,31 +34,31 @@ export class HydrogenSession {
     return new this(storage, session);
   }
 
-  has(key: string) {
-    return this.session.has(key);
+  get has() {
+    return this.#session.has;
   }
```

Comment:
By turning method wrappers into getters like this, we delegate the type to the actual session without losing information due to `key: string` (i.e. `string` is more generic that what the session expects).

Open to ideas for a simpler syntax that keeps the types right.